packages/cozy-harvest-lib/src/helpers/queries.js

```diff
@@ -61,3 +61,12 @@ export const buildAppsByIdQuery = appSlug => ({
     singleDocData: true
   }
 })
+
+export const buildCountTriggersQuery = () => ({
+  definition: Q('io.cozy.triggers')
+    .where({ worker: ['client', 'konnector'] })
+    .indexFields(['worker']),
+  options: {
+    as: 'io.cozy.triggers'
```

Comment:
```suggestion
    as: 'io.cozy.triggers/by_worker_client_konnector'
```

Reply:
This is not only way better... Cozy client use a deduplication system based on the id. 

Imagine this stupid code: 

```jsx
const [triggers1, triggers2] = Promise.all([client.query(Q('io.cozy.triggers'), { as : 'io.cozy.triggers'})), client.query(Q('io.cozy.triggers').where({worker: 'client'})]
```

Then, triggers1 and triggers2 will have the same result even though the request is not the same. Your data will be wrong.